client/src/pages/orders/components/order-employee/order-employee.tsx

```diff
@@ -0,0 +1,113 @@
+import * as React from 'react';
+import { Box, Typography, Avatar, Menu, MenuItem, Dialog, DialogTitle, List, ListItem, ListItemAvatar, ListItemText } from "@mui/material";
+import { grey, red } from "@mui/material/colors";
+import { OrderEmployeeProps } from "./order-employee-props";
+import PersonAddAltIcon from "@mui/icons-material/PersonAddAlt";
+import { useCookies } from 'react-cookie';
+import { AdminService } from 'services/admin/admin-service';
+import { User } from 'types';
+import { UserRoleUtils } from 'utils/user-role-utils';
+
+export function OrderEmployee(props: OrderEmployeeProps) {
+  const { employee, client, onChangeAssignment } = props;
+
+  const [anchorEl, setAnchorEl] = React.useState<null | HTMLElement>();
+  const [cookies, setCookie, removeCookie] = useCookies(["token", "role"]);
+  const [dialogOpened, setDialogOpened] = React.useState(false);
+  const [users, setUsers] = React.useState<User[]>([]);
+
+  const optionsOpened: boolean = Boolean(anchorEl);
+  const adminService: AdminService = new AdminService(cookies);
+
+
+
+
+  const openAssignWorkerDialog = (): void => {
+    adminService.getUsers()
+      .then(res => setUsers(res.filter(x => x.id != client?.id && (UserRoleUtils.isWorker(x.roles) || UserRoleUtils.isAdmin(x.roles)))))
+      .then(() => setDialogOpened(true))
+      .then(() => setAnchorEl(null));
+  }
+
+  
+  
+
```

Comment:
czemu te puste linie?

Reply:
Chciałem sobie rozdzielić niektóre elementy od siebie. W sensie: łatwiej mi się to odczytuje